docs/data/charts/radar/radar.md

```diff
@@ -93,17 +87,48 @@ Otherwise, the other area would catch the pointer event, making it impossible to
 
 To remove highlight, set the `highlight` prop to `'none'`.
 
-## Tooltip 🚧
+## Tooltip
+
+Like other charts, the radar chart [tooltip](/x/react-charts/tooltip/) can be customized with slots.
+The `trigger` prop of the `tooltip` slot accepts the following values:
 
-## Composition 🚧
+- `'axis'`—the user's mouse position is associated with a metric. The tooltip displays data about all series along this specific metric.
+- `'item'`—when the user's mouse hovers over a radar area, the tooltip displays data about this series.
+- `'none'`—disable the tooltip.
+
+{{"demo": "RadarTooltip.js" }}
+
+## Composition
 
 For composition, use the `RadarDataProvider` to provide `series` and `radar` props.
 
-Providing components for radar composition is still a work in progress.
-If you miss some element or explanation, please open an issue describing what you want to achieve, and what is missing.
+In the background, you can render the `RadarGrid` and `RadarMetricLabels` that will display the grid and the labels.
```

Comment:
What is meant by "in the background"? Are we trying to nudge users to render them as the first few children so that they're rendered below the other elements?

```suggestion
In the background, you can render the `RadarGrid` and `RadarMetricLabels` that display the grid and the labels.
```

Reply:
> Are we trying to nudge users to render them as the first few children so that they're rendered below the other elements

I did not realise it looks like a requirement..

I could go with something more descriptive like

```suggestion
The `RadarGrid` and `RadarMetricLabels` components render the grid and the labels.
```